internal/controller/isbservicerollout/isbservicerollout_progressive.go

```diff
@@ -123,7 +91,7 @@ func (r *ISBServiceRolloutReconciler) ProcessPromotedChildPreUpgrade(
 	return false, nil
 }
 
-func (r *ISBServiceRolloutReconciler) ProcessPromotedChildPostUpgrade(
+func (r *ISBServiceRolloutReconciler) ProcessPromotedChildPostFailure(
```

Comment:
I thought maybe this was a more accurate name for this case, since it's only the failure case and not the success case